cli/pack/docker.go

```diff
@@ -0,0 +1,107 @@
+package pack
+
+import (
+	_ "embed"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/otiai10/copy"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/docker"
+	"github.com/tarantool/tt/cli/templates/engines"
+)
+
+//go:embed templates/Dockerfile.pack.build
+var buildDockerfile []byte
+
+// RunInDocker runs tt pack in docker container.
+func RunInDocker(cmdCtx *cmdcontext.CmdCtx, packCtx *PackCtx,
```

Comment:
Please give more meaningful name to this function. PackInDocker?
RunInDocker says nothing about what is going to run in docker.

Reply:
FIxed.